model_servers/llamacpp_python/Makefile

```diff
@@ -12,10 +26,23 @@ mistral-7b-instruct-v0.1.Q4_K_M.gguf:
 install:
 	pip install -r tests/requirements.txt
 
-.PHONY: run
-run:
-	podman run -it -d -p 8001:8001 -v ./models:/locallm/models:ro,Z -e MODEL_PATH=models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=8001 --net=host ghcr.io/redhat-et/model_servers
+.PHONY: run-linux
+run-linux:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models:ro,Z -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) --net=host $(IMAGE); cd model_servers/llamacpp_python;
```

Comment:
I think we can combine all of these lines together and then check if we are running on linux or darwin and change them slightly.  

Make /locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf  a variable at the top.
I don't think the cd model_servers/llamacpp_python at the bottom does anything. since when the Makefile is done, you are back in the parent shell.